modules/tinymce/src/core/main/ts/fmt/Hooks.ts

```diff
@@ -17,7 +17,7 @@ import * as ArrUtils from '../util/ArrUtils';
  * @class tinymce.fmt.Hooks
  */
 
-const postProcessHooks = {}, filter = ArrUtils.filter, each = ArrUtils.each;
+const postProcessHooks: Record<string, Array<(editor: Editor) => void>> = {}, filter = ArrUtils.filter, each = ArrUtils.each;
```

Comment:
Given the amount of types here, should postProcessHooks maybe get its own line?

Reply:
Done, although it really doesn't matter given this file in general breaks our normal formatting rules.